update index, add embeds page, move nextra stuff

markflorkowski opened the pull request.

markflorkowski committed:
update index, add embeds page, move nextra stuff

Dax911 reviewed (approved):
Holy cow that was fast. And u formatted everything. Thanks.

Dax911 commented:
There is an issue w vercel and locating a specific import. Stand by for fixes.

Dax911 committed:
Update image.mdx

Missing resource screwed up deployment.  Will end up deleting these files anyway.

Dax911 committed:
Delete code-highlighting.mdx

Unneeded

Dax911 committed:
Delete meta.json

un needed
